dashboard/client/src/common/MultiTabLogViewer.tsx

```diff
@@ -30,17 +31,42 @@ export type MultiTabLogViewerTabDetails = {
 export type MultiTabLogViewerProps = {
   tabs: MultiTabLogViewerTabDetails[];
   otherLogsLink?: string;
+  /**
+   * If set, this multi-tab log viewer will remember the last selected tab and start on that tab
+   * the next time this component is rendered.
+   *
+   * Different string values to provide different contexts for this memory. For example, if you
+   * want all multi-tab log viewers in the actor detail page to share one memory, they should have
+   * the same string value here.
+   */
+  tabMemoryContextKey?: string;
```

Comment:
this feels too specific, maybe name it `pageKey`?

also i see this is optional, wouldn't this not work with
```typescript
    `MultiTabLogViewer-tabMemory-${tabMemoryContextKey}`,
```

Reply:
I'll just do `contextKey`.

useLocalStorage hook itself doesn't actually read or write from the localStorage, it returns functions that do. IN the case where `tabMemoryContextKey` is undefined, we never call those functions. This is because we can't conditionally call hooks in react. I'll add a comment about this.